LocalSettings.php

```diff
@@ -554,19 +554,18 @@
-			'Database exists (wiki still active)' => 'Wiki database name and subdomain already exist. Please visit the local wiki and contribute there. Please reach out to any local bureaucrat to request any permissions if you require them. If bureaucrats are not active on the wiki after a reasonable period of time, please start a local election and ask a Steward to evaluate it on the Stewards\' Noticeboard. Thanks.',
+			'Use Public Test Wiki' => 'Please use Public Test Wiki, https://publictestwiki.com, to test the administrator and bureaucrat tools (as well as Miraheze\'s MediaWiki configuration). You should review and follow all TestWiki:Policies, especially TestWiki:Testing policy and TestWiki:Main policy, reverting all tests you perform in the reverse order which you performed them. Request permissions at TestWiki:Request permissions. Thank you.',	
+			'Database exists (wiki still active)' => 'Wiki database name and subdomain already exist. Please visit the local wiki and contribute there. Please reach out to any local bureaucrat to request any permissions you may require. If bureaucrats are not active on the wiki after a reasonable period of time, please start a local election and ask a Steward to evaluate it on the Stewards\' Noticeboard. Thank you.',
 			'Database exists and the requester used to be a bureaucrat (wiki already deleted)' => 'Wiki database name already exists, but the wiki itself has already been deleted in accordance with the Dormancy Policy. I will request a Steward to undelete it for you. When it has been undeleted and reopened, please visit the local wiki and ensure you make at least one edit or log action every 45 days. Wikis are only deleted after 6 months of complete inactivity; if you require a Dormancy Policy exemption, you should review the policy and request it once your wiki has at least 40-60 content pages. Thank you.',
 			'Duplicate request' => 'Declining as a duplicate request, which needs more information. Please do not edit this request and instead go back into your original request. Also, please do not submit duplicate requests. Thank you.',
 			'Content Policy (unsubstantiated insult)' => 'Declining per Content Policy provision, "Miraheze does not host wikis with the sole purpose to spread unsubstantiated insult, hate or rumours against a person or group of people." Thank you for understanding.',
 			'Content Policy (makes it difficult for other wikis)' => 'Declining per Content Policy provision, "A wiki must not create problems which make it difficult for other wikis." Thank you for understanding.',	
-			'Duplicate wiki' => 'We already have a wiki that covers this topic. Please contribute to that wiki instead. Thank you.',
```

Comment:
It is probably a good idea to let this stay. I would prefer not to have duplicates of wikis with hundreds of content pages. I frequently decline wikis when I know there is a wiki that already covers the topic and has enough content.

Reply:
> It is probably a good idea to let this stay. I would prefer not to have duplicates of wikis with hundreds of content pages. I frequently decline wikis when I know there is a wiki that already covers the topic and has enough content.

Well, I did say this could be re-added, but it does need some changes, as for example, we _do_ allow many duplicate encyclopedia (whether specialized or broad, general purpose) wikis. Likewise, we wouldn't decline one user's wanting to have a wiki that describes what they view as the best websites, or the worst websites, despite there already being Reception wikis covering those fairly broad topics as subjectively determined. If they ended up simply copying and pasting those wikis verbatim, or nearly verbatim, that would not be okay, and Stewards would step in to mediate a solution if asked.

So, I'll re-add that reason, with the above modifications in mind.

Reply:
Changes were done as a separate PR (https://github.com/R4356th/mw-config/pull/3), even though I modified the same file in your `patch-5`. So, I would recommend just merging this pull request first, then merging the second one with the further changes. Thanks!

Reply:
I have reviewed that PR.